orion/sites/admin/src/programs/programEdit.js

```diff
@@ -178,9 +178,42 @@ export class ProgramEditPage extends React.Component {
 }
 
 function checkProgram(program) {
-  // *todo* change this later!
+  let checkPro = true;
+
+  let programId = program.programId;
+
+  // Checks if programId contains special characters
+  let invalidChars = "~`!#$%^&*+=-[]\\\';,/{}|\":<>?";
+  for (let i = 0; i < programId.length; i++) {
+    if (invalidChars.indexOf(programId.charAt(i)) != -1) {
+      checkPro = false;
```

Comment:
Sorry, I forgot. When something fails the validation checker, changing `checkPro` to false is good. But could you also change `errorMessage`? The idea is that when a program fails, we don't want to just have "BAD PROGRAM!" We want to also say why it failed.
Thanks!